### PR TITLE
Add a tooltip to FullscreenControl

### PR DESCRIPTION
**Pull request by dmytro-gokun**
As described in this feature proposal: https://github.com/mapbox/mapbox-gl-js/issues/8009

### CODE REVIEW

File: src/ui/control/fullscreen_control.js [hunk 1]
```diff
@@ -87,12 +87,18 @@ class FullscreenControl {
...
         const button = this._fullscreenButton = DOM.create('button', (`${this._className}-icon ${this._className}-fullscreen`), this._controlContainer);
-        button.setAttribute("aria-label", "Toggle fullscreen");
         button.type = 'button';
+        this._updateTitle();
         this._fullscreenButton.addEventListener('click', this._onClickFullscreen);
         window.document.addEventListener(this._fullscreenchange, this._changeIcon);
     }
 
+    _updateTitle() {
+        const title = this._isFullscreen() ? "Exit fullscreen mode" : "Enter fullscreen mode";
```
Comment (andrewharvey) [inline src/ui/control/fullscreen_control.js]: Would simply "Enter fullscreen" and "Exit fullscreen" suffice?

Comment (dmytro-gokun) [reply src/ui/control/fullscreen_control.js]: Yep, you are right. Fixed it.

**Comment by dmytro-gokun** [comment]
This is my first experience with open source, so it might be a stupid question to ask.. Is there any additional step(s) I have to take before this can be merged?

**Comment by dmytro-gokun** [comment]
@andrewharvey Thanks a lot for merging!

**Comment by andrewharvey** [comment]
Thanks for the PR. I think sometimes it won't be merged straight away just to given other contributors a chance to chime in with any comments before being merged.

**Comment by dmytro-gokun** [comment]
Gotcha. It's just not very clear when you do it for the first time. But now i understand that sometimes i just need to wait :)

**Pull request merged**